### ISSUE TITLE
do not loot ice pile twice for the same creature

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#115]
During death we can pick up both the creature and its spawned ice pile
Add logic to only loot the items once - although this seem to be no harm it's confusing and inefficient.

**Comment by SteveTownsend** [comment]
Test evidence after the fix of dual loot case
```
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(1035): [VERBOSE] {31556} (2020-07-09 04:05:23 PM) Process REFR 0xff0022c4 with base object Ice Pile/0x001069e4
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(124): [VERBOSE] {31556} (2020-07-09 04:05:23 PM) dynamic REFR 0xff0022c4 or base 0x001069e4 for Ice Pile
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(1105): [MESSAGE] {31556} (2020-07-09 04:05:23 PM) Got ash-pile REFR 0xff0022bc from REFR 0xff0022c4
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(411): [MESSAGE] {31556} (2020-07-09 04:05:23 PM) scanning container/body Ice Wraith/0xff0022bc
J:\GitHub\SmartHarvestSE\src\Utilities\debugs.cpp(155): [MESSAGE] {31556} (2020-07-09 04:05:23 PM) CONT 00023ab3 2b(43) [Ice Wraith]
J:\GitHub\SmartHarvestSE\src\Utilities\debugs.cpp(196): [MESSAGE] {31556} (2020-07-09 04:05:23 PM) ExtraContainerChanges 95041a5e [Dawnbreaker Hit] 00000164EFF77370 count=1 playable=0  - unknown
J:\GitHub\SmartHarvestSE\src\Utilities\debugs.cpp(196): [MESSAGE] {31556} (2020-07-09 04:05:23 PM) ExtraContainerChanges 0003ad6a [Ice Wraith Teeth] 0000016450502DD0 count=2 playable=1  - ingredient
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(534): [MESSAGE] {31556} (2020-07-09 04:05:23 PM) block looted container Ice Wraith/0xff0022bc
J:\GitHub\SmartHarvestSE\src\Data\iniSettings.cpp(82): [DEBUG] {31556} (2020-07-09 04:05:23 PM) Get config setting 2/2/ingredient = 1.000000
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(642): [MESSAGE] {31556} (2020-07-09 04:05:23 PM) get Ice Wraith Teeth (2) from container Ice Wraith/0xff0022bc
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(741): [VERBOSE] {31556} (2020-07-09 04:05:23 PM) Trigger glow for Ice Wraith/0xff0022bc
J:\GitHub\SmartHarvestSE\src\Looting\InventoryItem.cpp(25): [VERBOSE] {31556} (2020-07-09 04:05:23 PM) get Ice Wraith Teeth/0x0003ad6a (2)
J:\GitHub\SmartHarvestSE\src\Looting\InventoryItem.cpp(81): [VERBOSE] {31556} (2020-07-09 04:05:23 PM) Move item Ice Wraith Teeth (2)
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(1035): [VERBOSE] {31556} (2020-07-09 04:05:23 PM) Process REFR 0xff0022bc with base object Ice Wraith/0x00023ab3
J:\GitHub\SmartHarvestSE\src\Looting\objects.cpp(115): [DEBUG] {31556} (2020-07-09 04:05:23 PM) Actor is follower = false
J:\GitHub\SmartHarvestSE\src\Looting\objects.cpp(131): [DEBUG] {31556} (2020-07-09 04:05:23 PM) Actor summoned = false
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(124): [VERBOSE] {31556} (2020-07-09 04:05:23 PM) dynamic REFR 0xff0022bc or base 0x00023ab3 for Ice Wraith
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(1078): [MESSAGE] {31556} (2020-07-09 04:05:23 PM) Skip immediate-loot deadbody, already looted on this pass 0xff0022bc, base = Ice Wraith/0x00023ab3
```
Loose ice pile handling
```
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(1035): [VERBOSE] {31556} (2020-07-09 04:05:50 PM) Process REFR 0xff0022be with base object Ice Pile/0x001069e4
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(124): [VERBOSE] {31556} (2020-07-09 04:05:50 PM) dynamic REFR 0xff0022be or base 0x001069e4 for Ice Pile
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(1105): [MESSAGE] {31556} (2020-07-09 04:05:50 PM) Got ash-pile REFR 0xff001c07 from REFR 0xff0022be
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(411): [MESSAGE] {31556} (2020-07-09 04:05:50 PM) scanning container/body Ice Wraith/0xff001c07
J:\GitHub\SmartHarvestSE\src\Utilities\debugs.cpp(155): [MESSAGE] {31556} (2020-07-09 04:05:50 PM) CONT 00023ab3 2b(43) [Ice Wraith]
J:\GitHub\SmartHarvestSE\src\Utilities\debugs.cpp(196): [MESSAGE] {31556} (2020-07-09 04:05:50 PM) ExtraContainerChanges 95041a5e [Dawnbreaker Hit] 00000166484C42A0 count=1 playable=0  - unknown
J:\GitHub\SmartHarvestSE\src\Utilities\debugs.cpp(196): [MESSAGE] {31556} (2020-07-09 04:05:50 PM) ExtraContainerChanges 0003ad6a [Ice Wraith Teeth] 000001657ECCEB00 count=1 playable=1  - ingredient
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(534): [MESSAGE] {31556} (2020-07-09 04:05:50 PM) block looted container Ice Wraith/0xff001c07
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(642): [MESSAGE] {31556} (2020-07-09 04:05:50 PM) get Ice Wraith Teeth (1) from container Ice Wraith/0xff001c07
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(741): [VERBOSE] {31556} (2020-07-09 04:05:50 PM) Trigger glow for Ice Wraith/0xff001c07
J:\GitHub\SmartHarvestSE\src\Looting\InventoryItem.cpp(25): [VERBOSE] {31556} (2020-07-09 04:05:50 PM) get Ice Wraith Teeth/0x0003ad6a (1)
J:\GitHub\SmartHarvestSE\src\Looting\InventoryItem.cpp(81): [VERBOSE] {31556} (2020-07-09 04:05:50 PM) Move item Ice Wraith Teeth (1)
```

**Comment by SteveTownsend** [comment]
Fixed for v1.7.0.0
